PracticeGit/TestComplex.c

```diff
@@ -0,0 +1,102 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// ------------------------------------------------------------------------------------------------
+// Testomplex.c -  Program to perform arithmetic operations on complex numbers.
+// ------------------------------------------------------------------------------------------------
+#include <stdio.h>
+#include <math.h>
+#include "Complex.h"
+
+ // Function to print complex numbers
+void PrintComplex (ComplexNumber c);
+
+// Function to compare both values
+int CompareComplex (ComplexNumber result, float expectedReal, float expectedImg);
+
+// Function to run the test cases
+void RunTestCases ();
+
+void PrintComplex (ComplexNumber c) {
+   if (c.img >= 0) printf ("%.2f + %.2fi\n", c.real, c.img);
+   else printf ("%.2f - %.2fi\n", c.real, -c.img);
+}
+
+int CompareComplex (ComplexNumber result, float expectedReal, float expectedImg) {
+   return (result.real == expectedReal && result.img == expectedImg);
+}
+
+void RunTestCases () {
+   ComplexNumber input[] = { {3.0, 7.0}, {4.0, -3.0} };
+   float expected[] = { 7.00, 4.00, -1.00, 10.00, 33.00, 19.00, 7.62, 3.00, -7.00 };
+   printf ("Running Additon Test...\n");
+   ComplexNumber resultAdd = OpAdd (input[0], input[1]);
```

Comment:
You are testing everything here manually. This can be automated. 
1. Use a loop
2. Compute expected values